.github/workflows/Custom_ACT.yml

```diff
@@ -0,0 +1,111 @@
+name: Custom ACT workflow for nektos/act (Compiler for Github Workflows locally)
+
+on:
+  push:
+    tags:
+      - '*'
+#on:
+#  push:
+#    branches: [ develop ]
+#    # Sequence of patterns matched against refs/tags
+#    tags:
+#      - 'v*' # Push events matching v*, i.e. v1.0, v20.15.10
+#  pull_request:
+#    branches: [ develop ]
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  BUILD_TYPE: Release
+  Python_REQUIRED_VERSION: 3.8
+
+jobs:
+  linux_release:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+    permissions:
+      # Needed permission to upload the release asset
+      contents: write
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python ${{ env.Python_REQUIRED_VERSION }}
+      id: setup-python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ env.Python_REQUIRED_VERSION }}
+
+    - name: Setup QtIFW 4.x
+      uses: jmarrec/setup-qtifw@v1
+      with:
+        qtifw-version: '4.x'
+
+    - name: Install Python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install aqtinstall --user
+    - name: "Install Latest CMake"
+      shell: bash
+      run: |
+        sudo apt remove cmake
+        wget https://github.com/Kitware/CMake/releases/download/v3.24.1/cmake-3.24.1-linux-x86_64.sh
+        sudo mv cmake-3.24.1-linux-x86_64.sh /opt
+        chmod +x /opt/cmake-3.24.1-linux-x86_64.sh
+        sudo bash /opt/cmake-3.24.1-linux-x86_64.sh --skip-license --prefix=/root
+        sudo ln -sf /root/bin/* /usr/local/bin
+        ls /root/bin/
+        cmake --version 
```

Comment:
Right now we get away with not installing an updated CMake because the GitHub image already has 3.24 on it.  I wonder if we should add a bit of logic to ours to check if a recent version is installed and then install it if not.  That would get our stuff working with `act`.